accepted/0000-install-peer-deps.md

```diff
@@ -0,0 +1,144 @@
+# Install Peer Dependencies
+
+## Summary
+
+Install `peerDependencies` along with packages that peer-depend on them.
+
+Ensure that a validly matching peer dependency is found at or above the
```

Comment:
does this effectively mean that a nonzero `npm ls`, at least due to peer dep issues, will fail the install?

Reply:
Yes, it does. So, I guess a breaking change here is that some packages that can be installed today may not be installable? Maybe we could have a flag to not install peer deps. Hm.

Reply:
No, on the contrary, I think that's awesome. I think it's a massive mistake that `npm install` is allowed to succeed when `npm ls` would fail, and if that could be part of npm 7 I would be ecstatic! Tons of frequently reported bugs would vanish in all the ecosystems I participate in that use peer deps - eslint, babel, react, jest, to name a few.

Reply:
Isn't that enough? If the installation will fail on missing peer dependencies, that will force users to add the missing peers.